packages/amplify_datastore_plugin_interface/test/testData/Person.dart

```diff
@@ -69,34 +61,16 @@ class Person extends Model {
     }
   }
 
-  TemporalDateTime? get createdAt {
```

Comment:
What happened to `createdAt`/`updatedAt`?

Reply:
This is a custom type, should not contain model meta fields. I think the original generated code was outdated...